### ISSUE TITLE
[BUG] maketx --monochrome-detect should generate Y not R image for exr formats

### ISSUE THREAD

**Issue by ThiagoIze** [OpenImageIO/oiio#3204]
**Describe the bug**
The `maketx --monochrome-detect` generates a single channel texture when it's safe to do so. Currently, if the output format of the .tx file is an exr, it names this channel R instead of Y even though Y was previously established to be the OIIO single channel name in #1434.  I guess this happens because exr allows named channels, but in this case we never specified what this name should be, so R is incorrectly used. This is a minor problem for us because some DCCs will display these textures as red images instead of grey.

**To Reproduce**
run `maketx --monochrome-detect --format exr` on a 3 channel texture where all channels are the same. Then run `oiiotool --info -v your_texture.tx` and you'll see results such as:

```
> oiiotool.exe --info -v Abstract_Soft_Left_sm.exr
Reading Abstract_Soft_Left_sm.exr
Abstract_Soft_Left_sm.exr :  512 x  512, 3 channel, float openexr
    channel list: R, G, B

> oiiotool.exe --info -v Abstract_Soft_Left_sm.tx
Reading Abstract_Soft_Left_sm.tx
Abstract_Soft_Left_sm.tx :  512 x  512, 1 channel, float openexr
    MIP-map levels: 512x512 256x256 128x128 64x64 32x32 16x16 8x8 4x4 2x2 1x1
    channel list: R
```

**Expected behavior**
`channel list: R` should have specified Y.

**Evidence**
See above for an example.

**Platform information:**
This was tested with a forked branch of 2.3.2.dev on Windows.

**Comment by lgritz** [comment]
Easy fix! Stay tuned...

**Comment by lgritz** [comment]
Proposed fix: #3205
